test/verify/check-storage-stratis

```diff
@@ -41,6 +41,11 @@ def get_stratis_stop_type_opt(execute):
     return ""
 
 
+def create_pool_key(machine, keyname, passphrase):
+    # this is a bit complicated, see https://bugzilla.redhat.com/show_bug.cgi?id=2246923
```

Comment:
I don't actually think it's that complicated, and there is no bug worth fixing here. (Well, maybe the docs.) So I think this comment is misleading. It was a mistake to not use --keyfile-path from the start in our tests, since --capture-key is the mode for interactive use.

Reply:
Hm, if `--capture-key` is for interactive use, it is *highly* misleading. "Interactive" means "human on a tty" for me, and that is more robust than stdin. Piping in a password is usually "noninteractive"/automated in my book. But yes, we can discuss that in the bugzilla.